pkgs/servers/sql/cockroachdb/cockroachdb-bin.nix

```diff
@@ -32,6 +32,10 @@ buildFHSEnv {
 
   runScript = "${src}/cockroach";
 
+  extraInstallCommands = ''
+    cp -P $out/bin/cockroachdb $out/bin/cockroach
```

Comment:
```suggestion
   ln -s $out/bin/cockroachdb $out/bin/cockroach
```

Reply:
I also thought since `cockroachdb` already is a symlink that `cp` whould make more sense here

```
$ ls -l result/bin/
total 8
lrwxrwxrwx 1 root root 68  1. Jan 1970  cockroach -> /nix/store/cc5wh2wnh36fi6yh5impcffn67jgq3qk-cockroachdb-23.1.7-bwrap
lrwxrwxrwx 1 root root 68  1. Jan 1970  cockroachdb -> /nix/store/cc5wh2wnh36fi6yh5impcffn67jgq3qk-cockroachdb-23.1.7-bwrap
```

Reply:
Oh, right.